update/delta/delta_1.3.3_remove_old_views_and_triggers.sql

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Why is this a 1.3.3 delta, as opposed to 1.3.2? As far as I understand the current release is 1.3.1, so why not extending the current delta_1.3.2 file instead of creating a new one? This question is mainly for me to undertand…

Reply:
@elemoine Good point. 

1/ I think that if there is a 1.3.2 delta, so the version should be 1.3.2. So there's an error here. 
2/ Why not merging the DLETE in the 1.3.2, but the delta name will have a wrong name
3/ Agree for the FUNCTION, corrected.

Reply:
> 1/ I think that if there is a 1.3.2 delta, so the version should be 1.3.2. So there's an error here.

Yep, maybe. I am not familiar with the qWat release process. So I am not sure.



> 2/ Why not merging the DLETE in the 1.3.2, but the delta name will have a wrong name

1.3.2 is not released yet, so I tend to think that you can freely change the delta name. (I'd love that someone confirm this.)

Also, in my opinion, delta names don't make too much sense. If it was me I'd just numbers. For example: delta_1.3.2_1.sql, delta_1.3.2_2.sql, etc. For the record, I think Pum does support multiple delta files per version already, but I haven't verified it. I am not suggesting to switch from delta names to delta numbers right now, but maybe this is something that could be discussed at some point.